.circleci/config.yml

```diff
@@ -0,0 +1,104 @@
+
+version: 2
+jobs:
+  build:
+    docker:
+      - image: openjdk:8
+        environment:
+          CI_TERRIBLENESS: 30.seconds
+        parallelism: 2
+    working_directory: ~/linkerd
+    steps:
+      - checkout
+
+      - setup_remote_docker
+
+      - restore_cache:
+          key: linkerd-{{ checksum "project/LinkerdBuild.scala" }}-{{ checksum "admin/src/main/resources/io/buoyant/admin/package.json" }}
```

Comment:
this whole file could use some contents.  why are checksums based on these two files?  LinkerdBuild rarely changes...